opts.py

```diff
@@ -47,6 +47,9 @@ def model_opts(parser):
     parser.add_argument('-dec_layers', type=int, default=2,
                         help='Number of layers in the decoder')
 
+    parser.add_argument('-width', type=int, default=3,
```

Comment:
Can we call this `kernel_size` throughout. Think that is the pytorch standard name.

Reply:
the true kernel_size in the conv layer is (width, 1). I have added a sentence to explain the relation between width and kernel_size  and in the help